internal/stolonctl/pkg/defaults/defaults.go

```diff
@@ -31,4 +31,28 @@ const (
 
 	// ClusterName defines name of stolon cluster
 	ClusterName = "kube-stolon"
+
+	// Custom Resource Definition variables
+	// StolonUpgradeName defines CRD name
+	StolonUpgradeName = "stolonupgrades.stolon.gravitational.io"
+	// StolonUpgradeGroup defines CRD group
+	StolonUpgradeGroup = "stolon.gravitational.io"
+	// StolonUpgradeVersion defines CRD version
+	StolonUpgradeVersion = "v1"
+	// StolonUpgradeAPIVersion definec CRD API version
+	StolonUpgradeAPIVersion = "stolon.gravitational.io/v1"
+	// StolonUpgradeScope defines CRD scope
+	StolonUpgradeScope = "Namespaced"
+	// StolonUpgradeKind defines CRD kind
+	StolonUpgradeKind = "StolonUpgrade"
+	// StolonUpgradePlural defines CRD plural form
```

Comment:
Can you elaborate on when you need plural/singular forms? It might be helpful to the user